app/package.json

```diff
@@ -15,8 +15,8 @@
     "@babel/core": "^7.0.0-0",
     "@babel/plugin-proposal-optional-chaining": "^7.14.5",
     "@nuxtjs/axios": "^5.13.6",
-    "@nuxtjs/sentry": "^5.1.5",
-    "@sentry/tracing": "^7.62.0",
```

Comment:
Any idea what this is and why it needs to be removed?

Reply:
@sentry/tracing is deprecated and no longer needed from @nuxtjs/sentry v7.1.12
https://github.com/nuxt-community/sentry-module/releases/tag/v7.1.12